src/service/vaults.ts

```diff
@@ -74,7 +74,7 @@ const watchPriceFeeds = (prefix: string) => {
               .getState()
               .setPriceError(
                 brand,
-                new Error('Unexpected error, price feed missing'),
+                new Error('Unexpected error, price feed missing.'),
```

Comment:
For want of a dot, the hash was lost

Reply:
To be clear for posterity, a new hash was the [goal](https://github.com/Agoric/dapp-inter/issues/299#issuecomment-2147990966) (I couldn't find anywhere to put a nail 😄)